server/lib/parties/party-service.test.ts

```diff
@@ -82,33 +93,52 @@ describe('parties/party-service', () => {
     let leader: PartyUser
     let party: PartyRecord
 
-    test('should throw if inviting yourself', () => {
-      expect(() =>
+    test('should throw if inviting yourself', async () => {
+      await expect(() =>
```

Comment:
is the function form necessary on these? Seems like if it's a promise you can just pass the promise directly? (The point of the function is to give it something it can `try/catch` around)